Self-Study/Algorithms_Answers.md

```diff
@@ -33,7 +33,18 @@ c)  def bunnyEars(bunnies):  #went over this with Luisan in class :)
 
 #### Exercise II
 
-
+```Suppose that you have an _n_-story building and plenty of eggs. Suppose also
+that an egg gets broken if it is thrown off floor _f_ or higher, and doesn't get
+broken if dropped off a floor less than floor _f_. Devise a strategy to
+determine the value of _f_ such that the number of dropped eggs is minimized.
+
+Write out your proposed algorithm in plain English or pseudocode and give the
+runtime complexity of your solution.```
+
+-So I have a building that is n stories tall.
+-when egg is dropped from f floor or higher, it will break.
+-when egg is dropped from any floor less than f, it will not break.
```

Comment:
great explanation. yes, remember: every time you're reducing your array by half, think O(log n)